neoload/commands/run.py

```diff
@@ -94,8 +95,14 @@ def create_data(name, description, as_code, web_vu, sap_vu, citrix_vu, reservati
 
 def prepare_external_url(data, external_url, external_url_label):
     if external_url is not None:
-        data['externalUrl'] = external_url
-    if external_url_label is not None:
+        match = re.match(r'\[(.*)\]\((.*)\)', external_url)
```

Comment:
We could add a documentation in the option, as for the other command

Reply:
I'm thinking you can put in common with test_result.py in lib folder